app/src/main/java/com/randomappsinc/studentpicker/grouping/GroupMakingSettingsViewHolder.java

```diff
@@ -9,16 +11,21 @@
 
 import butterknife.BindView;
 import butterknife.ButterKnife;
+import butterknife.OnCheckedChanged;
 import butterknife.OnTextChanged;
 
 class GroupMakingSettingsViewHolder {
 
     @BindView(R.id.number_of_names_in_list) TextView numberOfNamesInList;
     @BindView(R.id.num_of_names_per_group) EditText namesPerGroup;
     @BindView(R.id.num_of_groups) EditText numGroups;
+    @BindView(R.id.groups_warning_message) TextView warningMessage;
+    @BindView(R.id.autocomplete) CheckBox autocomplete;
 
     private GroupMakingSettings settings;
     private MaterialDialog dialog;
+    private String namesPerGroupInput;
```

Comment:
In general, it's better to stay away from having member variables that reflect information that's in the UI (which is the source of truth to the user). It's extra work maintaining these variables and easy to create bugs where what the user sees doesn't match what the business logic sees and is trying to do. If we need to get these values, we should use read them straight from the EditText.